bahmnicore-omod/pom.xml

```diff
@@ -6,7 +6,7 @@
         <artifactId>bahmni</artifactId>
         <version>1.1.0</version>
     </parent>
-    <artifactId>bahmnicore-omod</artifactId>
+    <artifactId>bahmnicore-ipd-omod</artifactId>
```

Comment:
I think we need to have the updated artifactId for API pom as well. We have added trivy scanning too. Please have a look on this commit https://github.com/Bahmni/openmrs-module-bahmnicommons/commit/219fcb40bb970f898f35e877a60792246c609cd0

Reply:
Made the changes as suggested